src/screens/AccountUtxoList/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { useEffect } from "react";
+import { StyleSheet, View } from "react-native";
+
+import { NavigationProp } from "@react-navigation/native";
+
+import { Layout } from "../../styles";
+import { useAccountsContext } from "../../components/accounts/AccountsContext";
+import { Utxo } from "../../models/Utxo";
+import navUtils from "../../utils/NavUtils";
+
+import SelectedUtxosHeader from "../../components/accounts/SelectedUtxosHeader";
+
+import UtxoItem from "./components/UtxoItem";
+
+interface Props {
+  navigation: NavigationProp<any>;
+}
+
+export default function AccountUtxoListScreen({
+  navigation
+}: Props) {
+  const accountsContext = useAccountsContext();
+  const { currentAccount } = accountsContext;
```

Comment:
you can destructure objects in a single line:

```
const { currentAccount, utxos } = useAccountsContext();
```

Reply:
`utxos` is a property of `Account`, not of `AccountsContext`

Reply:
I just learned you can destructure nested objects, so yes single line like this works:

`  const { currentAccount, currentAccount: { utxos } } = useAccountsContext();`